src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -614,6 +617,59 @@ def _validate_params(self):
             )
             self._model_fuse_fn_name = "no_fuse"
 
+    def _log_quantization(
+        self,
+        module: Module,
+        epoch: float,
+        steps_per_epoch: int,
+    ):
+        """
+        Check whether to log an update for the learning rate of the modifier.
+
+        :param module: module to modify
+        :param optimizer: optimizer to modify
+        :param epoch: current epoch and progress within the current epoch
+        :param steps_per_epoch: number of steps taken within each epoch
+            (calculate batch number using this and epoch)
+        """
+
+        def _log(tag, value):
+            self.log_scalar(
+                tag=tag,
+                value=value,
+                epoch=epoch,
+                steps_per_epoch=steps_per_epoch,
+            )
```

Comment:
optional nit: you might not actually save any lines by having this helper function. there's 4 calls to log in here, which means it saves 8 lines. the function def is about 8 lines though